spec/jobs/daily_likes_digest_job_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+RSpec.describe DailyLikesDigestJob, type: :job do
+  it 'envia e-mail com resumo diário de curtidas' do
+    user = create(:user)
+    post_a = create(:post, user:)
+    post_b = create(:post, user:)
+    comment = create(:comment, user:)
+    3.times { create(:like, likeable: post_a) }
+    5.times { create(:like, likeable: post_b) }
+    2.times { create(:like, likeable: comment) }
+    mail = double('mail', deliver: true)
+    mailer = double('LikesMailer', notify_new_like: mail)
+    allow(LikesMailer).to receive(:with).and_return(mailer)
+    allow(mailer).to receive(:notify_like).and_return(mail)
+
+    DailyLikesDigestJob.perform_now(user:)
+
+    expect(mail).to have_received(:deliver).once
+  end
+
+  it 'não envia e-mail caso o usuário não tenha nenhuma curtida nas últimas 24 horas' do
+    user = create(:user)
+    travel_to(2.days.ago) do
+      post = create(:post, user:)
+      10.times { create(:like, likeable: post) }
+    end
+
+    mail = double('mail', deliver: true)
+    mailer = double('LikesMailer', notify_new_like: mail)
+    allow(LikesMailer).to receive(:with).and_return(mailer)
+    allow(mailer).to receive(:notify_like).and_return(mail)
+
```

Comment:
Pessoal, fiquei confuso com esse teste. Criar likes há dois dias atrás e testar que nada foi enviado, já que nenhum like foi recebido no ultimo dia, talvez seja o mesmo que não criar nada e esperar que nenhum email foi recebido, como por exemplo ` allow(Like).to receive(:where).with('created_at >= ?', 24.hours.ago).and_return([]) ` . 🍀